helpers.py

```diff
@@ -62,3 +63,21 @@ def get_cancellers(self) -> Set:
 
     def add_canceller(self, canceller: str) -> bool:
         return self.db.sadd(Key.CANCELLERS, canceller)
+    
+    def add_inv(self, author: int, inv: str):
+        player_inv = self.inv_builder(author)
+        return self.db.sadd(player_inv, inv)
+
+    def remove_inv(self, author, inv: str):
+        player_inv = self.inv_builder(author)
+        return self.db.srem(player_inv, inv)
+    
+    def get_inv(self, user: str):
```

Comment:
```suggestion
    def get_inv(self, user: str) -> Set:
```

We fetch and return a Set from the database.